sysbox-in-docker/sysbox

```diff
@@ -259,17 +243,32 @@ function maxs_setup() {
   sysctl -w kernel.keys.maxbytes=$((20*20000)) &> /dev/null
 }
 
+# Sysbox performs certain operations that require read-write access to sysfs. In
+# the scenario where this script executes (i.e. privileged containers), sysfs is
+# usually mounted in read-write mode, but there is one bug (apparently in
```

Comment:
I suspect the problem here is that when launching a privileged ** pod **, the pause container's spec has sysfs mounted read-only, so a similar mount shows up in the sibling container (somehow). I doubt the problem is a bug in containerd itself.  In other words, when launching a privileged pod, the sysfs mount is showing up as read-only instead of read-write.

Reply:
Could be, makes sense.